_data/publications/jimenez-rescue-hpc18.yml

```
@@ -0,0 +1,6 @@
+title: Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform
```

```suggestion
title: "Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform"
```

I believe you can't put a `:` in a string unless it's quoted (or not next to a space, IIRC, so YAML can support urls)

---

In general, long strings should probably be surrounded by quotes, with just short, simple strings plain.